src/css/profile/mobile/common/oneui-common.less

```diff
@@ -26,6 +26,7 @@
     --appbar-main-text-color: @appbar-main-text-color;
     --appbar-subtitle-color: @appbar-sub-title-color;
     --appbar-miltiline-title-color: @appbar-multiline-title-color;
+    --bottom-bar-color: @bottom-bar-color;
```

Comment:
How about using the @background-color directly without @bottom-bar-color?

Reply:
This color is different for light and dark color.

This is `in progress` PR, you don't have to do review 'in progress' PRs.

Reply:
Sorry, I didn't watch the `in progress` I will review it later.

Reply:
No problem... I just want to say that this is changing and there is nothing to waste of energy. Anyway, Thank you for review.

Reply:
But, I have a problem with:
`@bottom-bar-color: @background-color;`
The less doesn't include proper color for dark theme.
When I set in this place:
`@bottom-bar-color: #f2f2f2`
Everything is ok.
It seems like less.js change reference to variable of get it from cache.